courses/serializers/v1/courses.py

```diff
@@ -32,7 +32,6 @@ class CourseSerializer(BaseCourseSerializer):
     page = CoursePageSerializer(read_only=True)
     programs = serializers.SerializerMethodField()
 
-    @extend_schema_field(int)
     def get_next_run_id(self, instance) -> int | None:
```

Comment:
This change was the motivation for PR, but I noticed a few other nullable serializer method field annotations to update, too.